plio/models.py

```diff
@@ -158,3 +158,17 @@ def video_update_cache(sender, instance, **kwargs):
     plios = Plio.objects.filter(video_id=instance.id)
     # invalidate saved cache for the plios
     invalidate_cache_for_instances(plios)
+
+
+@receiver(post_save, sender=Item)
+@receiver(post_delete, sender=Item)
+def item_update_cache(sender, instance, **kwargs):
+    # invalidate saved cache for the plio
+    invalidate_cache_for_instance(instance.plio)
+
+
+@receiver(post_save, sender=Question)
+@receiver(post_delete, sender=Question)
+def question_update_cache(sender, instance, **kwargs):
+    # invalidate saved cache for the plio
+    invalidate_cache_for_instance(instance.item.plio)
```

Comment:
here, you'll have to invalidate the cache for `instance.item` too right?

Reply:
not really as we're not caching item. Only Plio responses which have items included

Reply:
And we're not caching item or question because we don't use any GET request for items or questions instances right?

Reply:
ah sorry - makes sense!

Reply:
@deepansh96 Yes, for now. Though we do have endpoints but I don't think we're using GET endpoints for item and question. In case we do, we can apply caching there as well.